apps/web/src/app/globals.css

```
@@ -929,6 +929,26 @@
+   behind hover").
+
+   Owning the resting state here rather than as a bare `opacity-0` is what
+   keeps this `!important`-free: Tailwind compiles `group-hover:opacity-100` to
+   `.group-hover\:opacity-100:is(…)` at specificity (0,2,0), which outranks this
+   (0,1,0) rule on its own merits — so the reveal wins no matter which order
+   Tailwind happens to emit the two in. */
+@utility reveal-on-hover {
+  opacity: 0;
+  transition: opacity 300ms;
+  @media (hover: none) {
+    opacity: 1;
```

Stylelint (17.14.0)

[error] Invalid position for declaration (no-invalid-position-declaration)

(no-invalid-position-declaration)